pytorch_lightning/trainer/optimizers.py

```diff
@@ -150,13 +150,16 @@ def configure_schedulers(
                 scheduler['reduce_on_plateau'] = isinstance(
                     scheduler['scheduler'], optim.lr_scheduler.ReduceLROnPlateau
                 )
-                if scheduler['reduce_on_plateau'] and scheduler.get('monitor', None) is None:
+                if scheduler['reduce_on_plateau'] and scheduler.get('monitor', None) is None \
+                        and not is_manual_optimization:
                     raise MisconfigurationException(
                         'The lr scheduler dict must include a monitor when a `ReduceLROnPlateau` scheduler is used.'
                         ' For example: {"optimizer": optimizer, "lr_scheduler":'
                         ' {"scheduler": scheduler, "monitor": "your_loss"}}'
                     )
                 lr_schedulers.append({**default_config, **scheduler})
+            elif isinstance(scheduler, optim.lr_scheduler.ReduceLROnPlateau) and is_manual_optimization:
+                lr_schedulers.append({**default_config, 'scheduler': scheduler})
             elif isinstance(scheduler, optim.lr_scheduler.ReduceLROnPlateau):
```

Comment:
logic is correct but maybe combine both of them into one?

Reply:
Ok, resolved in https://github.com/PyTorchLightning/pytorch-lightning/pull/7643/commits/6a7e3009785851f220d706dc523118fb7046e2be.

However, also invalid keys are required in pytorch lightning, replacing invalid with default parameters.